### ISSUE TITLE
How to remove alpha channel?

### ISSUE THREAD

**Issue by stone12379** [lovell/sharp#1248]

**Comment by lovell** [comment]
Hello, did you see the [flatten](http://sharp.pixelplumbing.com/en/stable/api-operation/#flatten) operation?

**Comment by stone12379** [comment]
Yes, I saw that. But the problem is that when applying flatten to a CMYK image which is supposed to be converted to sRGB(JPG), something unwanted showed in the final image. So I would like to know if there's a way to remove alpha channel totally without merging it to the final image.

**Comment by lovell** [comment]
Thanks for the additional information, there's nothing currently to do exactly what you need so perhaps a future possible `removeAlpha` operation would be a good idea.

What input format are you using that supports CMYK+A? TIFF? It would make a good test case.

**Comment by lovell** [comment]
Commit c14434f adds this feature, will be in v0.20.6. Thanks for the suggestion.

**Comment by stone12379** [comment]
Fantastic! Thank you, lovell!

**Comment by lovell** [comment]
v0.20.6 now available.
